server.js

```diff
@@ -9,4 +10,31 @@ const app = express();
 app.use(express.json());
 app.use(cors());
 
+app.post('/login', async (req, res) => {
```

Comment:
Not bad for the first time. Try splitting stuff into files like `routes.js` that will include `auth,js` and `projects.js` each with their own prefixes. Also, check the correct usage of `express` through `http` module. It's minor, but implemented as well.